boot/bootutil/src/bootutil_misc.c

```diff
@@ -837,3 +837,28 @@ boot_set_confirmed(void)
     flash_area_close(fap);
     return rc;
 }
+
+/**
+ * Populates the version information of the
+ * currently installed primary application
+ *
+ * @param[in] version Destination version structure buffer
+ * @return 0 on success; nonzero on failure.
+ */
+int boot_get_current_version(struct image_version *version)
+{
+    struct boot_loader_state state; // Dummy state
+    struct boot_status status;      // Dummy status
```

Comment:
We don't use C++-style line comments. Also these comments are not very relevant so they don't need to be here.